content/concepts/nft.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Data NFTs
+description: Ocean Protocol uses ERC721 NFTs to represent the intellectual property (IP) of published datasets or algorithms, and to further sub-license access to it with user roles or ERC20 data tokens.
+---
+
+A non-fungible token stored on the blockchain represents a unique asset. NFTs can represent images, videos, digital art, or any piece of information. NFTs can be traded and allow transfer of ownership and copyright. [EIP-721](https://eips.ethereum.org/EIPS/eip-721) defines an interface for handling NFTs on Ethereum blockchain. The creator of the NFT can deploy a new contract on Ethereum or any Blockchain supporting NFT related interface and also, transfer the ownership through transfer transactions.
+
+## Terminology
+
+- **Base IP** means the artifact being copyrighted. Represented by the {ERC721 address, tokenId} from the publish transactions.
+- **Base IP holder** means the holder of the Base IP. Represented as the the actor that did the initial "publish" action.
+- **Sub-licensee** is the holder of the sub-license. Represented as the entity that controls address ERC721.\_owners[tokenId=x].
+- **To Publish**: Claim copyright or exclusive base license
+- **To Sub-license**:Transfer one (of many) sub-licenses to new licensee: ERC20.transfer(to=licensee, value=1.0)
```

Comment:
all this talk in here about "sub-licenses" but we never mention datatokens? I changed this in the overall page description at top and I strongly suggest if sub-licensing or managing access to a dataset with data tokens is meant with "sub-licensing", then it should be made way more clear throughout whole document

Reply:
Agreed. Click on the "Files changed" tab and you'll see specific recommendations that I'm making on exactly this.